addons/l10n_generic_coa/demo/account_invoice_demo.xml

```diff
@@ -82,6 +82,38 @@
         </record>
 
         <!-- ==== Post invoices ==== -->
+        <function model="account.move" name="_onchange_partner_id" context="{'check_move_validity': False}">
```

Comment:
please add comments on the reasons of these onchange calls + why with check_move_validity=False in context

Reply:
Better now?